monkey/monkey_island/cc/ui/src/components/Main.tsx

```diff
@@ -165,42 +191,57 @@ class AppComponent extends AuthComponent {
   }
 
   render() {
+    const {availablePlugins, installedPlugins, numberOfPluginsThatRequiresUpdate} = this.state;
```

Comment:
Is there a way that we can move this state to a separate component so that it doesn't intermingle with everything else here?

Reply:
We can try